content-repo/extra-docs/releases/20.10.0.md

```diff
@@ -0,0 +1,969 @@
+- **RiskIQ Asset Type**
+- **RiskIQ Skip Manual Tasks**
+- **RiskIQ Support Contact**
+#### IncidentTypes
+- **RiskIQ Asset Management**
+#### IndicatorFields
+##### RiskIQAsset Added To Inventory  
+Date and Time when that asset was added to RiskIQ Digital Footprint inventory.
+##### RiskIQAsset Brands  
+Name(s) of the brand(s) applied to the asset.
+##### RiskIQAsset Confidence  
+Discovery confidence level of the asset.
+##### RiskIQAsset CVEs  
+CVEs detected from the details of the RiskIQ Asset.
+##### RiskIQAsset Enterprise Asset  
+Has the asset been designated as an enterprise asset by RiskIQ Digital Footprint.
+##### RiskIQAsset First Seen  
+Date and Time when the asset was first observed on RiskIQ Digital Footprint.
+##### RiskIQAsset Inventory Status  
+Inventory status of the asset.
+##### RiskIQAsset Last Seen  
+Date and Time when the asset was most recently observed on RiskIQ Digital Footprint.
+##### RiskIQAsset Last Updated  
+Date and Time when the most recent update was performed on the asset by a user action on RiskIQ Digital Footprint.
+##### RiskIQAsset Organizations  
+Name(s) of the organization(s) applied to the asset.
+##### RiskIQAsset Priority  
+Priority of the asset.
+##### RiskIQAsset Tags  
+Name(s) of the tag(s) applied to the asset.
+##### RiskIQAsset Type  
+The type of the asset.
+##### RiskIQAsset UUID  
+Unique identifier of the asset on RiskIQ Digital Footprint.
+#### IndicatorTypes
+**Packs/RiskIQDigitalFootprint/IndicatorTypes/reputation-RiskIQAsset.json**
+#### Integrations
+##### RiskIQ Digital Footprint  
+The RiskIQ Digital Footprint integration enables your security team to manage assets outside your firewall. Using the integration, you can view asset details, add or update assets and analyze your digital footprint from the adversary's perspective.
+#### Layouts
+- **RiskIQ Asset Management - Summary**
+- **RiskIQ Asset Management - New/Edit**
+- **RiskIQ Asset Management - Mobile**
+- **RiskIQ Asset Management - Quick View**
+#### Playbooks
+##### Enrich Incident With Asset Details - RiskIQ Digital Footprint  
+Enriches the incident with asset details, and enriches the asset with the incident URL on the RiskIQ Digital Footprint platform. This playbook also sends an email containing the owner's information to the primary or secondary contact of the asset and provides the user with an opportunity to update or remove the asset.
+Supported integration: RiskIQ Digital Footprint
+##### Update Or Remove Assets - RiskIQ Digital Footprint  
+Using various user inputs, this playbook checks if the user wants to update or remove an asset, and performs the respective actions.
+Supported integration: RiskIQ Digital Footprint
+
+
+---
+
+### New: Spamcop Pack v1.0.0
+#### Integrations
+##### Spamcop  
+SpamCop is an email spam reporting service, integration allow checking the reputation of an IP address
+
+
+---
+
+### New: Troubleshoot Pack v1.0.0
+#### Scripts
+##### CertificatesTroubleshoot  
```

Comment:
Add a description for the script

Reply:
Exports all custom certificate-related information from the Python Docker container and decode it using RFC. In addition, it will get the certificate located in the specified endpoint.

Reply:
Looks good.